point_viewer_grpc_proto_rust/build.rs

```diff
@@ -48,12 +48,15 @@ fn main() {
     println!("cargo:rerun-if-changed=src/proto.proto");
 
     let git_repo_root = find_git_repo_root();
+    let old_path = env::var("PATH");
```

Comment:
Would it make sense to have a small RAII helper for temporarily changing the path and use it everywhere, something like?
```rust
struct TempPath {
    old_path: String,
}

impl TempPath {
    pub fn new(new_path: &Str) -> Self {
        let old_path = env::var("PATH");
        env::set_var("PATH", new_path);
        TempPath{ old_path }
    }
}

impl Drop for TempPath {
    fn drop(&mut self) {
        env::set_var("PATH", self.old_path.unwrap_or("".to_string()));
    }
}
```

Reply:
Something like that, yes. I'd do that in a separate `build_proto` crate that we can use in build scripts for other crates.

Reply:
Cool, thanks